src/components/Calculator.css

```diff
@@ -0,0 +1,34 @@
+#calculator {
+  width: 33%;
+  padding: 1rem;
+  background-color: #2a2a2a;
+  display: grid;
+  gap: 3px 3px;
+  grid-template: repeat(5, 1fr) / repeat(4, 1fr);
+}
+
+.display {
+  display: flex;
+  grid-area: 1 / 1 / span 1 / span 4;
+  background-color: #fff;
+  padding: 1rem;
+  font-size: 2rem;
+  justify-content: flex-end;
+}
+
+button {
+  border: 1px solid #fff;
+  font-size: 2rem;
+}
+
+.btn-orange {
+  background-color: rgb(250, 125, 41);
+}
+
+.btn-gray {
+  background-color: rgb(241, 238, 238);
+}
+
+.zero {
+  grid-area: 6 / 1 / span 1 / span 2;
+}
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/158434063-453e9dfc-c2ae-4a62-a25c-3019a768cf6d.png)

Therefore:

- [ ] Please, kindly match the given design on your project.

Your Implementation | Target design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/158440045-82e3ca78-3aa8-4f47-b7b6-ea7c5cb648cf.png) | ![image](https://user-images.githubusercontent.com/57421823/158434325-bac86723-032f-4c3d-8738-95b13cea447e.png)

Hint:  Focus on the background colors, and the transparency of the lines 💪 
Hint 2:  You could install a Firefox or Chrome extension "Color picker" to know exactly the colors used on the design.